.travis.yml

```diff
@@ -14,7 +14,12 @@ jdk:
   - openjdk7
   - oraclejdk8
   - oraclejdk9
-matrix:
+stages:
+  - name: test
+  - name: check
+  - name: deploy
+    if: branch = master
```

Comment:
We need to test if this is enough to suppress pull requests from attempting a deploy. I'm just not sure.

I'm not worried about bad jars being deployed, because Travis CI makes sure pull requests don't inherit your environment variable settings, therefore won't be able to publish jars to your clojars account... but what I don't want is for every pull request to be labeled "Checks have failed" because of the deploy step.